src/Mailgun/Model/Suppression/Unsubscribe/Unsubscribe.php

```diff
@@ -66,6 +118,24 @@ public function getAddress()
     }
 
     /**
+     * Set the email address unsubscribed
+     * 
+     * @param string $address The email address unsubscribed
+     */
+    private function setAddress($address = NULL)
```

Comment:
Same here, why is this optional?

Reply:
Yes, since you could be interested in init empty object or even "clear" the object or attribute after received the data

Reply:
I don't think it should be optional, based on the API docs